Index.html

```diff
@@ -77,5 +77,170 @@ <h1>I'm Kerlos Baha.<br />Glad to see <br />you!</h1>
       </section>
       <span class="shape"> </span>
     </main>
+    <div class="space"></div>
+
+    <!-- first work page -->
+
+    <section class="repeated-section">
+      <img class="repeated-img" src="/picture/Snapshoot Portfolio.png" alt="" />
+      <p class="repeated-p1">Tonic</p>
```

Comment:
Kindly update your project title to use a `<hx>` tag🙏. This is for semantic reasons, as this text represents the title of the project.

Reply:
Fixed.